src/rendering/measure.ts

```diff
@@ -144,10 +144,8 @@ export class Measure {
       const isLastMeasureEntry = measureEntryIndex === opts.measureEntries.length - 1;
 
       if (measureEntry instanceof StaveSignature) {
-        staveSignature = measureEntry;
```

Comment:
The bug was that we reassigned `staveSignature` before creating the next `MeasureFragment` with it below.